tests/unit/jest.config.js

```diff
@@ -23,6 +34,7 @@ module.exports = {
     'docs/public/',
     'docs/static',
   ],
+  ...ignoreWebComponentTests,
```

Comment:
i tried adding this rule inside the original `testPathIgnorePatterns` array but the terminal would error out. this was the only configuration i could come up with that gave me the results i wanted.